### DISCUSSION TITLE
Watertabledepth behavior with potential as the boundary condition

### DISCUSSION THREAD

**Discussion by KeLiu7** [APSIMInitiative/ApsimX#9281] [Q&A] [answered]
Hi all, 

Why does the groundwater level with the `Potential` boundary condition only last for a limited time, instead of throughout the entire growing period? What causes it to switch to free drainage (i.e. red frame periods) later on?

@hut104 @sno036 @zur003  Could you please have a look? Many thanks😊

![image](https://github.com/user-attachments/assets/4fb2a31e-2022-465b-a395-85ce0b5c425a)
[swim3.zip](https://github.com/user-attachments/files/16858742/swim3.zip)

**Comment by hut104** [comment, marked as answer]
The crop roots are entering your bottom layer and it is no longer saturated there and so the effective water table depth is the bottom of the profile (which is the default if it goes below the depth of the profile.
Note that you swim simulation has only 7 layers.  I'd suggest adding a thickness object to set up a normal swim node structure.

**Reply by KeLiu7** [reply]
Many thanks Neil. Generally, how many thin layers are needed for a normal swimming node structure at least?

**Comment by hut104** [comment]
Basically, where there are strong gradients - near the surface, at strong texture contrasts or where you have saturated conditions next to dry.  And, as you've seen, have nodes below the root zone if you want to maintain a water table accurately.
________________________________
